src/styles/theme.ts

```diff
@@ -0,0 +1,36 @@
+import { createMuiTheme, responsiveFontSizes } from '@material-ui/core/styles';
+
+//TODO: Review and redo colors.
+const gravityColors = {
+  primary: '#4d4848',
+  secondary: '#008d7c',
+  primaryText: '#ffffff',
+  secondaryText: '#ffffff',
+  background: '#ffffff',
+};
+
+let theme = createMuiTheme({
+  typography: {
+    htmlFontSize: 10, //62,5%
+  },
+  palette: {
+    primary: {
+      main: gravityColors.primary,
+      contrastText: gravityColors.primaryText,
```

Comment:
If this refers only to text colour, I believe it should go after line 29

Reply:
It refers to text that is inside of components that use primary palette, it is different from the line 29 text property. You can find more details in material UI documentation.